nextflow.config

```
@@ -16,16 +16,17 @@ params {
   max_peptide_length = (mhc_class == 1) ? 11 : 16
   min_peptide_length = (mhc_class == 1) ? 8 : 15
   tools = 'syfpeithi'
-  genome = 'GRCh37'
-  input = false
+  genome_version = 'GRCh37'
   wild_type = false
   peptides_split_maxchunks = 100
   peptides_split_minchunksize = 5000
 
   // Additional annotation files
   proteome = false
 
-  // Boilerplate options  
+  // Boilerplate options
+  input = ''
```

```suggestion
  input = false
```
should be `false` if not set, is checked in `main.nf`

---

`''` evaluates to `false`

---

Ah OK good to know, but could it maybe still be used consistently? Either using `''` for all non-boolean variables (and other input parameters) or sticking to `false`?